# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/cairo-quartz.h
# Commit: dbb07ebff6b6
# Full Hash: dbb07ebff6b6d0f08ac91fdd2c989c705b016e43
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2021-04-29 09:26:05
# Regressor Bug: 739096
# File Overlap Count: 1
# Description:
#   Bug 739096 - Remove support for CGLayer-backed cairo quartz surface. r=jrmuizel
#   
#   This was a performance optimization that we no longer care about.
#   Removing it will avoid the need to merge the additions into the new
#   cairo-quartz-surface code from upstream.
# ==============================================================================

diff --git a/gfx/cairo/cairo/src/cairo-quartz.h b/gfx/cairo/cairo/src/cairo-quartz.h
--- a/gfx/cairo/cairo/src/cairo-quartz.h
+++ b/gfx/cairo/cairo/src/cairo-quartz.h
@@ -1,11 +1,11 @@
 /* cairo - a vector graphics library with display and print output
  *
- * Copyright � 2006, 2007 Mozilla Corporation
+ * Copyright � 2006, 2007 Mozilla Corporation
  *
  * This library is free software; you can redistribute it and/or
  * modify it either under the terms of the GNU Lesser General Public
  * License version 2.1 as published by the Free Software Foundation
  * (the "LGPL") or, at your option, under the terms of the Mozilla
  * Public License Version 1.1 (the "MPL"). If you do not alter this
  * notice, a recipient may use your version of this file under either
  * the MPL or the LGPL.
@@ -58,22 +58,16 @@ cairo_quartz_surface_create (cairo_forma
 cairo_public cairo_surface_t *
 cairo_quartz_surface_create_for_data (unsigned char *data,
 				      cairo_format_t format,
 				      unsigned int width,
 				      unsigned int height,
 				      unsigned int stride);
 
 cairo_public cairo_surface_t *
-cairo_quartz_surface_create_cg_layer (cairo_surface_t *surface,
-                                      cairo_content_t content,
-                                      unsigned int width,
-                                      unsigned int height);
-
-cairo_public cairo_surface_t *
 cairo_quartz_surface_create_for_cg_context (CGContextRef cgContext,
                                             unsigned int width,
                                             unsigned int height);
 
 cairo_public CGContextRef
 cairo_quartz_surface_get_cg_context (cairo_surface_t *surface);
 
 cairo_public CGContextRef